authentik/enterprise/apps.py

```diff
@@ -32,3 +32,9 @@ def check_enabled(self):
         from authentik.enterprise.models import LicenseKey
```

Comment:
So above this I copied the `lru_cache` in the tenancy PR, but we should remove it